Yoast/YoastSEO.js
# Skip images and empty divs when searching for keywords in the first paragraph

## Summary

This PR can be summarized in the following changelog entry:

* Improves keyword recognition in the first paragraph on texts which start with images and captions

## Relevant technical choices:

* Skips images and empty divs in the beginning of the text to provide correct feedback in the keywordInFirstParagraph assessment

## Test instructions

This PR can be tested by following these steps:

* Enter a text that starts with an image or a caption with an image or a division with an image (and nothing else), while the keyphrase appears in the first _real_ paragraph. Check if the keyphrase is recognized correctly.
* Use the following (or similar) patterns

```
<img class=“size-medium wp-image-801” src=“https://test.com/text.jpg” alt=“Some alttag” width=“300” height=“173” />

The text with the keyword. 
```

```
<div style="text-align: center;"><img src="https://test.com/test.jpg" alt="Some alttag" /></div>
<div>The text with the keyword. </div>
```

```
<div style="text-align: center;"><a><img src="https://test.com/test.jpg" alt="Some alttag" /></a></div><div>The text with the keyword. </div>
```

* Watch out! After copying html from here into the local environment there might appear extra symbols that do not allow proper analysis. Make sure to remove them before testing.

Fixes #1380
